packages/ozone/src/mod-service/index.ts

```diff
@@ -1451,6 +1451,25 @@ export class ModerationService {
     // Convert map values to an array and return
     return Array.from(statsMap.values())
   }
+
+  async getAccountTimeline(did: string) {
+    const result = await this.db.db
+      .selectFrom('moderation_event')
+      .where('subjectDid', '=', did)
+      .select([
+        sql<string>`TO_CHAR(CAST(${sql.ref('createdAt')} AS TIMESTAMP), 'YYYY-MM-DD')`.as(
```

Comment:
Perhaps following the same logic used in typescript to extract the date would be a nice way to keep things consistent:
```suggestion
        sql<string>`SPLIT_PART(${sql.ref('createdAt')}, 'T', 1)`.as(
```


Could even keep these utils next to each other:
```ts
const dateFromDatetime = (date: Date) => {
  const [date] = date.toISOString().split('T')
  return date
}

const dateFromDbDatetime = (dateRef: DbRef) => {
  return sql<string>`SPLIT_PART(${dateRef}, 'T', 1)`;
}
```